install_config/upgrading/manual_upgrades.adoc

```diff
@@ -899,6 +899,114 @@ For Pacemaker HA multiple master deployments:
 After the service restarts, the certificate update is complete.
 // end::30to31updatingcerts[]
 
+[[manual-upgrading-logging-components]]
+== Upgrading Logging Components
+
+If you have previously link:../../install_config/aggregate_logging.html[deployed
+logging components] and want to upgrade your EFK stack with new images, you must
+take the following steps to safely upgrade with minimal disruption to your log
+data.
```

Comment:
Might rename it "Upgrading the EFK Logging Stack" (looks like we use that phrasing in https://docs.openshift.org/latest/install_config/aggregate_logging.html#overview). I was wanting to get the word "logging" more obviously into the heading.